ro_help/hub/admin.py

```diff
@@ -102,8 +101,17 @@ def get_readonly_fields(self, request, obj=None):
     @transaction.atomic
     def save_model(self, request, ngo, form, change):
         super().save_model(request, ngo, form, change)
+
         if ngo.accepts_transfer:
-            if not ngo.accounts.all():
+            no_accounts = request.POST.get("accounts-TOTAL_FORMS", 0)
```

Comment:
Let's have a separate function that takes `request.POST` and `int(no_accounts)`.
```python
def has_accounts(self, request, no_accounts):
    for account_id in range(no_accounts):
        if f"accounts-{account_id}-DELETE" in request:
            continue

        if request.get(f"accounts-{account_id}-iban") and request.get(f"accounts-{account_id}-bank"):
            return True

    return False

Reply:
Maybe `account_idx` or `idx` instead of `account_id`.